guacamole/src/main/frontend/src/app/clipboard/directives/guacClipboard.js

```diff
@@ -40,13 +40,31 @@ angular.module('clipboard').directive('guacClipboard', ['$injector',
     var config = {
         restrict    : 'E',
         replace     : true,
+        scope: {
+            /**
+             * The message that will be displayed within the Clipboard Editor
+             * when it does not have focus.
+             *
+             * @type String
+             */
+            inactiveContent : '=',
```

Comment:
It's a little weird for the clipboard element to have to have the parent tell it what its own inactive text should be. This functionality should really be internal to the clipboard directive.

Reply:
Previously I had the translator injected into this and would retrieve the text from there, but I was asked to instead just pass the text from the client. This sounds like you are asking me to go back to the way the code originally was. Am I understanding that correctly?

Reply:
What I meant was to have the translation done in the template, as opposed to the controller. It _is_ technically done in a template now, but it should be the clipboard template, not the client template, and doesn't need to be passed through any controllers or javascript code at all.